src/psyclone/domain/lfric/lfric_extract_driver_creator.py

```diff
@@ -621,22 +627,19 @@ def _create_read_in_code(self, program, psy_data, original_symbol_table,
 
     # -------------------------------------------------------------------------
     @staticmethod
-    def _import_modules(program, sched):
+    def _import_modules(symbol_table, sched):
         '''This function adds all the import statements required for the
-        actual kernel calls. It finds all calls in the PSyIR tree and
-        checks for calls with a ImportInterface. Any such call will
-        get a ContainerSymbol added for the module, and a RoutineSymbol
-        with an import interface pointing to this module.
-
-        :param program: the PSyIR Routine to which any code must \
-            be added. It also contains the symbol table to be used.
-        :type program: :py:class:`psyclone.psyir.nodes.Routine`
-        :param sched: the schedule that will be called by the driver \
-            program created.
+        actual kernel calls. It finds all calls in the schedule and
+        checks for calls with an ImportInterface. Any such call will
+        add a ContainerSymbol for the module and a RoutineSymbol (pointing
+        to the container) to the symbol table.
+
+        :param symbol_table: the symbol table to which the symbols are added.
+        :type program: :py:class:`psyclone.psyir.symbols.symbol_table`
```

Comment:
s/program/symbol_table/
and /.symbol_table/.SymbolTable/

Reply:
Oops.